pages/users.tsx

```diff
@@ -1,6 +1,89 @@
+import React from "react";
+import { inter } from "../styles/fonts";
+import { useFetchUsers } from "./hooks/useFetchUsers";
+import { IoPersonOutline } from "react-icons/io5";
+import * as MaterialDesign from "react-icons/md";
+import { BiPhone } from "react-icons/bi";
 
-function Users() {
-  return <div>Users</div>;
-}
+type Address = {
+  street: string;
+  city: string;
+  country: string;
+
+  // This defines an object where keys are strings and values are also strings.
+};
+
+type User = {
+  id: number;
+  name: string;
+  email: string;
+  username: string;
+  phone: number;
+  address: Address; // Now the User type includes an address property.
+};
+
+const UsersList: React.FC = () => {
```

Comment:
```suggestion
const UsersList = () => {
```

The type is unnecessary. You have no props and Typescript would just infer that you are returning a React element